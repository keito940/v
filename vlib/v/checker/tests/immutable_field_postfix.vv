struct A {
	i int
}

fn main() {
	mut a := A{}
	a.i++
	a.i--
}
