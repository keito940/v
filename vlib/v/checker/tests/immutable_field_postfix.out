vlib/v/checker/tests/immutable_field_postfix.v:7:4: error: field `i` of struct `A` is immutable
    5| fn main() {
    6|     mut a := A{}
    7|     a.i++
             ^
    8|     a.i--
    9| }
vlib/v/checker/tests/immutable_field_postfix.v:8:4: error: field `i` of struct `A` is immutable
    6|     mut a := A{}
    7|     a.i++
    8|     a.i--
             ^
    9| }
